api/tests/Feature/UserTest.php

```diff
@@ -1812,7 +1813,7 @@ public function testCountApplicantsQuery(): void
         );
         $response->assertJson([
             'data' => [
-                'countApplicants' => 14, // including base admin user
+                'countApplicants' => 15, // including base admin user
```

Comment:
Yikes, this test is going to be hard to debug. Pre-existing problem, though.  Could you please add a comment breaking down who the 15 are?

Reply:
Same with the 10 below?